spec/abci/abci++_app_requirements.md

```diff
@@ -601,6 +601,13 @@ This is enforced by the consensus algorithm.
 This implies a maximum transaction size that is this `MaxBytes`, less the expected size of
 the header, the validator set, and any included evidence in the block.
 
+The Application should be aware that honest validators _may_ produce and
+broadcast blocks with up the configured `MaxBytes` size.
```

Comment:
```suggestion
broadcast blocks with up to the configured `MaxBytes` size.
```

Reply:
I caught that in the backport PR but not here 🤦. Thanks @glnro